src/main/io/serial_cli.c

```diff
@@ -500,6 +500,10 @@ static const char * const lookupTableLowpassType[] = {
     "NORMAL", "HIGH"
 };
 
+static const char * const lookupTableLedAux[] = {
+    "OFF", "1","2","3","4"
```

Comment:
To support the full range of available aux channels I think this will be better without a lookup table. The instead the parameter should be limited between 0 and MAX_AUX_CHANNELS.